### ISSUE TITLE
Update PandExo to V1.4 

### ISSUE THREAD

**Issue by natashabatalha** [ExoCTK/exoctk#301]
Kevin and I have pushed the V1.4 PandExo release. It includes several updates to the website front end, fixed broken links, updates to code, exomast queries and bunch of other stuff. 

Make sure you download the new pandeia reference data (v1p4) because the old data is not compatible with the new code. 

Also, you will want to add the exoctk google analytics before you install (so make sure you clone and install from git instead of pip) 

Lastly, make sure you test both the HST and the JWST calculations. Specifically, make sure that calling a planet through ExoMAST and inputting in fixed values both work and yield similar results for the same planet. Kevin and I have tested locally but we should double check.

**Comment by natashabatalha** [comment]
Great! What is the approximate timeline for this change to be implemented? I am getting increasing # of emails.

**Comment by kevin218** [comment]
This task is priority #1.  Let's get the website updated yesterday!

**Comment by bourque** [comment]
Sorry for letting this slip through the cracks.  I'll get on it!

**Comment by bourque** [comment]
I should ping `stargate` for this once its ready to be tested

**Comment by bourque** [comment]
@kevin218 @natashabatalha I believe I have the new version of `pandexo` deployed on `tlexo` now! 
 Thank you for the guidance @natashabatalha, it was quite helpful!

**Comment by natashabatalha** [comment]
Great!! I don't have access to `tlexo`. @kevin218 or @hrwakeford can you make sure both Webby and HST are working?

**Comment by bourque** [comment]
Update:  I tried to deploy `pandexo` version 1.4 to `plexo` (note that we can't actually test the deployment on `tlexo` for various reasons), however we got a `502` error when trying to access the service.  Furthermore, we don't actually have read access to the web server error logs, so it is not apparent what the problem may be.

We will have to ask IT to show us what is going on in the error logs and/or try to get things running locally and see if any obvious errors appear.
